client/components/AssignmentListContainer/index.jsx

```diff
@@ -193,6 +215,7 @@ const mapDispatchToProps = (dispatch) => ({
             filterBy, searchQuery, orderByField, orderDirection, filterByState,
             filterByType, filterByPriority
         )),
+    fetchMyAssignmentsCount: (filterByState) => dispatch(actions.assignments.ui.myAssignments(filterByState)),
```

Comment:
This dispatch could be changed to `() => dispatch(actions.assignments.ui.myAssignments(['assigned', 'submitted']))` and remove `getMyAssignmentsCount` method from the class and use `this.props.fetchMyAssignmentsCount` instead